components/gfx/display_list/mod.rs

```diff
@@ -681,6 +708,18 @@ impl ClippingRegion {
     }
 }
 
+impl SizeOf for ClippingRegion {
+    fn size_of_excluding_self(&self) -> usize {
+        self.complex.size_of_excluding_self()
+    }
+}
+
+impl SizeOf for ComplexClippingRegion {
+    fn size_of_excluding_self(&self) -> usize {
+        0
```

Comment:
Not needed because ComplexClippingRegion doesn't have anything hanging off it (i.e. no Box fields).